### ISSUE TITLE
How to fix the problem with "Segmentation fault (core dumped)" 

### ISSUE THREAD

**Issue by jiashuncheng** [MarcoMeter/recurrent-ppo-truncated-bptt#13]
After running the loss.backward() in line 293 of trainer.py, there emerges a problem with the "segmentation fault". Please, how to solve this problem?
